client/src/Views/Reports.vue

```diff
@@ -1,36 +1,110 @@
 <template>
   <div class="reports-container">
-    <h2 @click="createHeaders(recentShouts[0])">Reports</h2>
-    <Table :data="recentShouts" />
+    <h2 @click="createHeaders(shoutsToDisplay[0])">Reports</h2>
+    <v-tabs
+      fixed-tabs
+      background-color="green lighten-1"
+      light
+      >
+      <v-tab 
+        >Monthly Reports</v-tab>
+      <v-tab
+        >Users</v-tab>
+      <v-tab
+        >All Shouts</v-tab>
+    </v-tabs>
+    <div class="reports-container__main">
+      <v-date-picker
+        v-model="picker"
+        type="month"
+        color="green darken-1"
+        >
+      </v-date-picker>
+      <Table class="reports-container__table" :data="shoutsToDisplay" @rankView="toggleView" :type="rankView" :dateObj="this.createDateObj()"/>
+    </div>
   </div>
 </template>
 
 <script>
-import { getAllShouts } from "../apiCall";
-import { shoutsFormatter } from "../shared/formatters";
+import { getRankedByMonth } from "../apiCall";
+import { rankedShoutersFormatter} from "../shared/formatters";
 import Table from "../components/Table";
 
 export default {
   name: "Reports",
   components: { Table },
   data() {
     return {
-      recentShouts: [],
+      shouts: [],
+      shoutsToDisplay: [],
+      currentView: 'monthly',
+      picker: new Date().toISOString().substr(0, 7),
+      rankView: true,
     };
   },
   computed: {
     headers() {
-      return this.createHeaders(this.recentShouts);
+      return this.createHeaders(this.shoutsToDisplay);
     },
     formattedShoutouts() {
       return 1;
     },
   },
   async created() {
-    this.recentShouts = shoutsFormatter(await getAllShouts());
+    const { selectedMonth, selectedYear } = this.createDateObj();
+    const formattedShouts = await getRankedByMonth('shouter', selectedMonth, selectedYear);
+    this.shoutsToDisplay = rankedShoutersFormatter(formattedShouts);
   },
+  methods: {
+    changeReport: function (view) {
```

Comment:
This changeReport isn't being used anywhere. Is it for future use?

Reply:
I was thinking of using this to update the property that will eventually update what gets showed in the tabs, but forgot I had this in there. I'll get rid of it for now.